pkg/core/function.go

```diff
@@ -534,19 +530,17 @@ func (c *client) UpdateFunction(buildpackBuilder Builder, options UpdateFunction
 	if build == nil {
 		// function was built locally, attempt to reconstruct configuration
 		localBuild := BuildOptions{
-			RunImage:       annotations[buildpackRunImageAnnotation],
-			BuildpackImage: annotations[buildpackBuildImageAnnotation],
-			LocalPath:      appDir,
-			Artifact:       annotations[functionArtifactAnnotation],
-			Handler:        annotations[functionHandlerAnnotation],
-			Invoker:        annotations[functionOverrideAnnotation],
+			LocalPath: appDir,
```

Comment:
Don't we still want to save which actual image was used?

Reply:
I don't think so, that would add drift between the current images and the images when the function was initially created. We don't save those images for on-cluster builds.

If we ever end up with multiple builders then we'll need some way to pick which one to use, but we don't have that problem today.

Reply:
Fair point